packages/core/admin/ee/admin/hooks/useLicenseLimits/tests/index.test.js

```diff
@@ -0,0 +1,30 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { useFetchClient } from '@strapi/helper-plugin';
+import { useQuery } from 'react-query';
+import useLicenseLimits from '..';
+
+jest.mock('@strapi/helper-plugin', () => ({
+  useFetchClient: jest.fn(() => ({
+    get: jest.fn(),
+  })),
+}));
+
+jest.mock('react-query', () => ({
+  useQuery: jest.fn(),
+}));
+
+describe('useLicenseLimits', () => {
```

Comment:
i'd do a test to mock the server returning 500 and see how the hook responds.

Reply:
Good idea. Right now the hook does not handle errors at all and I'm wondering what should happen in the FE if it fails (e.g. because a customer blocks the endpoint).

Reply:
The hook responds as `useQuery` would respond. In this case it returns the license object and the data attribute on it is undefined in the case of an error, thus forfeiting the whole process. @gu-stav you added `license?.data` checks yourself 👍